src/libarena/lib.rs

```diff
@@ -393,9 +392,7 @@ struct TypedArenaChunk<T> {
     next: *mut TypedArenaChunk<T>,
 
     /// The number of elements that this chunk can hold.
-    capacity: usize,
-
-    // Objects follow here, suitably aligned.
+    capacity: usize, // Objects follow here, suitably aligned.
```

Comment:
Is there an issue for this? Mangling any commented out lines at the bottom of a struct might be a bummer in some cases.